javascript/collapsible.js

```diff
@@ -18,3 +18,7 @@ function collapseAllGroups() {
       tableRows[i].style.display = 'none';
   }
 }
+$("thead").find("th").on("click", function() {
```

Comment:
This code registers toggle-callbacks for all pages which load `collapsible.js` right now. Is that as expected and does it work for all tables on pages not related to comparing revisions? If it works and doesn't break anything else, let's keep it. If it doesn't work, the tables you want to collapse should get an additional CSS-class like `collapsible` and `$("thead")` should become something like `$("table.collapsible thead")` instead.

Reply:
This only works on the comparison revision. It doesnt seem to work on anything else. The reason might be that the comparison view is using a different css style than the rest of the templates.

Reply:
But you are not checking any CSS-style when applying the click-handler, aren't you? :-) You only test for HTML-elements and `thead` and `th` are available in multiple tables. The template for `BlueGrey` contains the following HTML:

    <table class="comparison">

So please add a CSS class for those tables which should be toggled like I suggested before:

    <table class="comparison collapsible">
    $("table.collapsible thead")[...]

This makes sure that we don't interfere with unrelated tables by accident and your changes can be used for other tables as well.

Reply:
Ahh okay now i understand. Yes this absolutely makes sense. I have added the collapsible class to each table where JS is to be applied. In all the three templates. Also modified the JS now to trigger when class of "collapsible" is clicked. :)